exercises/wordy/example.tt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
```

Comment:
I know that there was some discussion about removing this in #295. Should it be included in newly generated tests? I wasn't too sure whether a decision had been made 😄